openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -280,7 +280,7 @@ export const getVersionPath = (
 export const getTableTabPath = (tableFQN: string, tab = 'schema') => {
   let path = ROUTES.TABLE_DETAILS_WITH_TAB;
   path = path
-    .replace(PLACEHOLDER_ROUTE_TABLE_FQN, tableFQN)
+    .replace(PLACEHOLDER_ROUTE_TABLE_FQN, encodeURIComponent(tableFQN))
```

Comment:
Can we use [`getEncodedFqn`](https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-ui/src/main/resources/ui/src/utils/StringsUtils.ts#L131) util here?